draft_theme.theme

```diff
@@ -38,3 +38,15 @@ function draft_theme_preprocess_views_view_unformatted(&$variables) {
 //  }
 
 }
+
+/**
+ * Implements hook_preprocess_input().
+ *
+ */
+function draft_theme_preprocess_input(&$variables) {
+  // Use <button> element instead of <input type="submit" />
+  // @todo Remove this when added to Drupal core. https://www.drupal.org/node/1671190
```

Comment:
Use `@see` doc tag

Reply:
Link to issue should be inside the '@todo' - https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#todo